common/workunit/workunit.cpp

```diff
@@ -14024,23 +14024,52 @@ bool executeGraphOnLingeringThor(IConstWorkUnit &workunit, const char *graphName
 
 static void setResources(StringBuffer &jobYaml, const IConstWorkUnit *workunit, const char *process)
 {
+    class GenerateResources
```

Comment:
Feels like you are hand-rolling a lambda here - perhaps a lambda would be clearer?

Reply:
Actually I'm not sure what this code is doing. Took me a couple of reads to work out how it compiled!